### PR TITLE
fix some errors under bugsmash ineffectual assignment 

### PR DESCRIPTION
**Pull request by soniasingla**

### CODE REVIEW

**Review by markyjackson-taulia — CHANGES_REQUESTED** [review]
Also would like to understand why the changes in the file permissions

File: cmd/tink-worker/internal/worker.go [hunk 1]
```diff
@@ -473,7 +473,7 @@ func sendUpdate(ctx context.Context, logger log.Logger, client pb.WorkflowServic
 }
 
 func openDataFile(wfDir string, l log.Logger) *os.File {
-	f, err := os.OpenFile(wfDir+string(os.PathSeparator)+dataFile, os.O_RDWR|os.O_CREATE, 0644)
```
[the remark this answers is not in the document]
Comment (soniasingla) [reply cmd/tink-worker/internal/worker.go:476]: @parauliya musedev bot is saying it needs to be changed

[the remark this answers is not in the document]
Comment (soniasingla) [reply cmd/tink-worker/internal/worker.go:476]: because of gosec error

**Comment by soniasingla** [comment]
> Also would like to understand why the changes in the file permissions

musedev bot is saying it needs to be changed because of gosec error under  golint

**Comment by gauravgahlot** [comment]
> > Also would like to understand why the changes in the file permissions
> 
> musedev bot is saying it needs to be changed because of gosec error under golint

How did you test the changes?

**Comment by gauravgahlot** [comment]
@soniasingla How did you test these changes?

**Comment by Raj-Dharwadkar** [comment]
@soniasingla Please can you rebase and address the comments ?

**Comment by soniasingla** [comment]
@Raj-Dharwadkar closing this PR since the golint errors are not valid, thanks for checking :)

**Pull request closed without merging**